docs/research/user-experience/xmr-btc-user-interface-validation.md

```diff
@@ -113,3 +113,23 @@ Through user interviews we plan to collect more information on the current setup
 Specific prototypes showcase specific answers to the questions listed above. We may use the prototypes in interviews to showcase scenarios.
 
 
+## Feedback 
+
+### Possible Features List
+
+This section points out features that were mentioned by the community. These features will be evaluated and prioritized before we start building.
+
+#### Avoid receiving tainted Bitcoin
```

Comment:
Proposal: add a link to the feature request/comment so that we can refer back to it and ask more questions or even keep them in the loop if we implement it.

Reply:
Going in the direction that @bonomat mentioned: https://www.reddit.com/r/Monero/comments/k14hug/how_would_an_atomic_swap_ui_look_like/gdplnt8?utm_source=share&utm_medium=web2x&context=3